.appveyor.yml

```diff
@@ -1,36 +1,38 @@
-build: off
-version: dev-{build}
+build: false
 shallow_clone: false
 clone_folder: C:\projects\php-cs-fixer
 
 environment:
     matrix:
         - php_ver: 5.6.17
+          SKIP_LINT_TEST_CASES: 1
         - php_ver: 7.0.9
 
 cache:
-    - '%APPDATA%\Composer'
+    - '%LOCALAPPDATA%\Composer\files'
```

Comment:
`files` are cached libraries, but `repo` is which libraries and how to get them. without `repo` the `repo` will be first generated from scratch, and then reuse `files`. why not to cache `repo` as well? (basically, cache whole `Composer` dir)